security/src/main/resources/example_security.json

```diff
@@ -0,0 +1,221 @@
+{
+  "roles": [
```

Comment:
Can we pare this json down to only the permissions used by this example? E.g. I don't think anything is using clusterDeploy.